### PR TITLE
adding global SAR access through microsoft planetary compute

### PR DESCRIPTION
**Pull request by abradley60**
### Proposed changes
Include a brief description of the changes being proposed, and why they are necessary.

### Closes issues (optional)
- Closes Issue #000

### Checklist 
(Replace `[ ]` with `[x]` to check off)

- [x] Notebook created using the [DEA-notebooks template](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop)
- [x] Remove any unused Python packages from `Load packages`
- [x] Remove any unused/empty code cells
- [x] Remove any guidance cells (e.g. `General advice`)
- [x] Ensure that all code cells follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/) for code. The `jupyterlab_code_formatter` tool can be used to format code cells to a consistent style: select each code cell, then click `Edit` and then one of the `Apply X Formatter` options (`YAPF` or `Black` are recommended).
- [x] Include relevant tags in the final notebook cell (refer to the [DEA Tags Index](https://knowledge.dea.ga.gov.au/genindex/), and re-use tags if possible)
- [x] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated
- [ ] Test notebook on both the `NCI` and `DEA Sandbox` (flag if not working as part of PR and ask for help to solve if needed)
- [x] If applicable, update the `Notebook currently compatible with the NCI|DEA Sandbox environment only` line below the notebook title to reflect the environments the notebook is compatible with
- [x] Check for any spelling mistakes using the DEA Sandbox's built-in spellchecker (double click on markdown cells then right-click on pink highlighted words). For example:

sandbox_spellchecker

### CODE REVIEW

**Comment by abradley60** [comment]
@geoscience-aman this has both of our changes. Can you confirm it looks good? Also, will this be cloned down to users sandbox's on login during the session? Or does another branch get clones?

**Comment by geoscience-aman** [comment]
> @geoscience-aman this has both of our changes. Can you confirm it looks good? Also, will this be cloned down to users sandbox's on login during the session? Or does another branch get clones?

Has my changes and looks good to me! I have made a commit with minor formatting improvements. 

I think it's the `stable` branch that gets cloned. Can you please confirm @robbibt?

**Comment by robbibt** [comment]
Hey @geoscience-aman @abradley60, the Sandbox sync comes from `stable` - once this PR is merged into `develop`, we can aim to merge `develop` into `stable` ASAP so it will sync automatically!

File: Real_world_examples/Radar_water_detection.ipynb [hunk 1]
```diff
@@ -2309,11 +2309,663 @@
    "metadata": {},
```
Comment (robbibt) [inline Real_world_examples/Radar_water_detection.ipynb:2309]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Search the STAC Catalog
</pre><p>No need for this; only required for datacube access</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1263/discussion/#comment-1741342910'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Comment (robbibt) [inline Real_world_examples/Radar_water_detection.ipynb:2309]: <pre class="ql-syntax" spellcheck="false">Line #5.        #bands=["vv"],
</pre><p>Maybe remove?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1263/discussion/#comment-1741342915'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Comment (robbibt) [inline Real_world_examples/Radar_water_detection.ipynb:2309]: <pre class="ql-syntax" spellcheck="false">Line #2.    ds_s1["vh_dB"] = 10 * np.log10(ds_s1.vh)
</pre><p>I think <a href="https://github.com/geoscience-aman" target="_blank">@geoscience-aman</a> hada function for applying this step? If so, could be good to use that here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1263/discussion/#comment-1741342916'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

**Comment by robbibt** [comment]
Hey @abradley60, left some tiny comments, if you can fix these up I can approve and we can merge this in!

**Review by robbibt — APPROVED** [review]
Looks great!

**Pull request merged**